src/index.tsx

```diff
@@ -39,13 +39,27 @@ const ProfiledApp = Sentry.withProfiler(FeatureFlagApp)
 
 const history = createBrowserHistory()
 
+const TOO_MANY_REQUESTS_ERROR_CODE = 429
+
 setupSentry({ history })
 
 const queryClient = new QueryClient({
   defaultOptions: {
     queries: {
       suspense: true,
-      retry: 3,
+      retry: (failureCount, error) => {
+        // Do not retry if the response status is 429
+        if (
+          typeof error === 'object' &&
+          error !== null &&
+          'status' in error &&
+          error.status === TOO_MANY_REQUESTS_ERROR_CODE
+        ) {
+          return false
+        }
+        // Otherwise, retry up to 3 times
+        return failureCount < 3
```

Comment:
Does failure count start at 0 or 1?

Reply:
Failure count starts at 0 for the initial failure, then each retry it gets incremented 1 time

so first retry is at 1, then 2, then 3, then exit function

Reply:
Alrighty perfect, wasn't sure if they'd be like "Oh this failed once ... `failureCount === 1` off the bat